settings.ts

```diff
@@ -0,0 +1,32 @@
+/** Constants */
```

Comment:
Reference for use of settings.ts file: https://github.com/blacksmithgu/obsidian-dataview/blob/master/src/settings.ts